Makefile

```diff
@@ -1,22 +1,21 @@
 docstyle:
 	echo "Checking documentation with pydocstyle..."
-	python -m pydocstyle ${SRC_DIR}/pymc4/
+	pydocstyle pymc4/
 	echo "Success!"
 
 
 format:
 	echo "Checking code style with black..."
-	python -m black -l 100 --check pymc4/
+	black --check pymc4/
 	echo "Success!"
 
-
 style:
 	echo "Checking code style with pylint..."
-	python -m pylint pymc4/
+	pylint pymc4/
 	echo "Success!"
 
 black:
-	black -l 100 .
+	black pymc4/
```

Comment:
Does this cause a large change in the codebase? I know that most of the code we've written so far hasn't really touched the 100 limit, but looking at some of the NUTS code in PyMC3, I think that the dark underbellies of the samplers will really need the extra line length to be readable...

Reply:
I personally prefer having more than 80 lines...

Reply:
80 lines?! 😛

Reply:
its still 100. Black reads the file linked below. Removing the flag here because of Do not repeat yourself principles

https://github.com/pymc-devs/pymc4/blob/master/pyproject.toml

Reply:
TravisCI passing is evidence! Codebase is unchanged in this PR but we still get the CI pass :)

Reply:
Ah right! Forgot about the config file :smile: